test/main.sh

```diff
@@ -176,7 +176,7 @@ TEST_DIR="$(mktemp -d -t lxd-test.tmp.XXXX)"
 chmod +x "${TEST_DIR}"
 
 # Verify the dir chain is accessible for other users
-INACCESSIBLE_DIRS="$(namei -m "${TEST_DIR}" | awk '/^ d/ {print $1}' | grep -v 'x$' || true)"
+INACCESSIBLE_DIRS="$(namei -m "${TEST_DIR}" | awk '/^ d/ {print $1}' | grep -v '[xt]$' || true)"
```

Comment:
What is this fixing? Can we add a comment explaining the cryptic command please?

Reply:
This is mostly a guard rail for if the `sudo chmod +x ~` is not done.